docs/src/pages/components/css-baseline/css-baseline.md

```diff
@@ -94,6 +94,8 @@ Be aware, however, that using this utility (and customizing `-webkit-scrollbar`)
 - Set the font-weight to `theme.typography.fontWeightBold` for the `<b>` and `<strong>` elements.
 - Custom font-smoothing is enabled for better display of the Roboto font.
 
+Note that if you want to add additional `@font-face` declarations, you need to use the string CSS template syntax for adding style overrides, so that the previosly defined `@font-face` declarations won't be replaced. See the [typography's customization example](/customization/typography/#self-hosted-fonts) for more info.
```

Comment:
I'm not sure this part is relevant under h1 CssBasline > h2 Approach. It seems specific to the customization. I think that having it under https://deploy-preview-25583--material-ui.netlify.app/customization/typography/#self-hosted-fonts would be better.

Reply:
I had the same doubt, I was thinking as it is relevant to the `CssBaseline` component it would be better if it is on this page, but I see your point. I will move it to the `typography` customization page.

Reply:
Moved.